bms/bms_app/static/bms_app/css/create_prodStyle.css

```diff
@@ -0,0 +1,28 @@
+
+.createprod
```

Comment:
Follow CSS naming conventions, and write reusable CSS, for reference check below link.
https://blog.toughbyte.com/blabla-9fd86eae4e6c